packages/angular/src/stories/alert/alert.stories.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc.
+ * All Rights ReserveThis software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrAlertModule } from '@clr/angular';
+import { moduleMetadata } from '@storybook/angular';
+import { boolean, select } from '@storybook/addon-knobs';
+import { withDesign } from 'storybook-addon-designs';
+
+const basicTemplate = require('!!raw-loader!./basic.html');
+
+export default {
+  title: 'Alert',
+  decorators: [
+    withDesign,
+    moduleMetadata({
+      imports: [ClrAlertModule],
+    }),
+  ],
+  parameters: {
+    design: [
```

Comment:
Personally, I don't think we need the Figma files listed in Storybook. We could link off to them from our documentation pages though.